### ISSUE TITLE
Force two-person releases

### ISSUE THREAD

**Issue by scbedd** [Azure/azure-sdk-tools#4712]
If you queue a build, you cannot approve it for release.

Should we make this change? @weshaggard @kurtzeborn

**Comment by kurtzeborn** [comment]
Yes!  On the same day we enable this, either you or I should send a broad mail (and teams post) to notify the team of this change.
